esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -311,22 +424,34 @@ void setup() {
   httpUpdateServer.setup(&webServer);
 
   webServer.on("/all", HTTP_GET, []() {
-    String json = getFieldsJson(fields, fieldCount);
+    String json = getFieldsJson();
+    webServer.sendHeader("Access-Control-Allow-Origin", "*");
+    webServer.send(200, "application/json", json);
+  });
+  
+  webServer.on("/product", HTTP_GET, []() {
+    String json = "{\"productName\":\"" PRODUCT_FRIENDLY_NAME "\"}";
```

Comment:
TODO? Add additional compile-time constant values, including compile-time string of compilation date/time?  Since it's compile-time constant, would not add any discernable overhead to this callback.